storage/lib/src/main/java/org/calyxos/backup/storage/scanner/MediaScanner.kt

```diff
@@ -57,15 +58,18 @@ public class MediaScanner(context: Context) {
 
     internal fun scanMediaUri(uri: Uri, extraQuery: String? = null): List<MediaFile> {
         val extras = Bundle().apply {
-            val query = StringBuilder()
+            val query = StringBuilder().apply {
```

Comment:
Tests?

Reply:
we don't have tests for this part, because we'd need a phone image with known files to test against. maybe we can make a ticket for you to provide these files as part of the image or just let me know where they are? (still can't run the isntrumentation tests locally)